tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -927,6 +930,27 @@ def setUp(self):
         else:
             self.cleanup_gaia(full_reset=True)
 
+        # if video capture is enabled, then start recording
+        if self.capture:
+            self.start_video_capture()
+
+    # saves the captured video in /sdcard/ folder (only logical choice)
+    # trigger screenrecord command as a background process since this does not return until explicitly killed
+    # (this is why os.system() was used instead of device.manager.shell(),
+    # since shell() cannot start a background process
+    def start_video_capture(self):
+        self.capture_file = '%s_%s.mp4' % (self.methodName, datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S'))
+        os.system("adb shell screenrecord " + os.path.join("/sdcard", self.capture_file) + "&")
+
+    # stops the capture and pulls the video file
+    def stop_video_capture(self):
+        # killing the phone process with SIGINT, otherwise the video file will get corrupted
+        self.device.manager.killProcess("screenrecord", sig = 2)
```

Comment:
We can abstract this function in [DeviceManagerADB](https://dxr.mozilla.org/mozilla-central/rev/6020a4cb41a77a09484c24a5875bb221714c0e6a/testing/mozbase/mozdevice/mozdevice/devicemanagerADB.py). Let's create 2 new functions there:
``` py
def start_video_recording(self):
    # ...

def stop_video_recording(self):
    # ...
```

Then, we can hide the "SIGINT" detail there. Gaiatest wants just to stop the device, no matters how it's done.